sources/PanelButton.cpp

```diff
@@ -68,6 +73,14 @@ PanelButton::CreatePanel(int8 which, const BView* target, uint32 flavors,
 		panel = new BFilePanel(B_OPEN_PANEL, &msgr, NULL, flavors, false, NULL,
 			filter);
 
-		panel->Window()->SetTitle(pairs[which].locale);
+		BString title = B_TRANSLATE("Filer");
+		title += ": ";
+
+		if (prefix != NULL)
+			title << prefix << ' ';
```

Comment:
Oh, almost forgot... :)
You should do the "ReplaceFirst() trick" with a variable here. We don't know where the 'pairs[which].locale' appears in other languages.